bundles/org.openhab.binding.zm/src/main/java/org/openhab/binding/zm/internal/handler/ZmBridgeHandler.java

```diff
@@ -0,0 +1,645 @@
+        parameters.add(String.format("mode=%s", streamType));
+        parameters.add(String.format("monitor=%s", id));
+        return buildUrlWithParameters("/cgi-bin/zms", parameters);
+    }
+
+    private String buildUrl(String path) {
+        return buildUrlWithParameters(path, EMPTY_LIST);
+    }
+
+    private String buildUrlWithParameters(String path, List<String> parameters) {
+        StringBuilder sb = buildBaseUrl(path);
+        String joiner = "?";
+        for (String parameter : parameters) {
+            sb.append(joiner).append(parameter);
+            joiner = "&";
+        }
+        if (zmAuth.usingAuthorization()) {
+            sb.append(joiner).append("token=").append(zmAuth.getAccessToken());
+        }
+        return sb.toString();
+    }
+
+    private StringBuilder buildBaseUrl(String path) {
+        StringBuilder sb = new StringBuilder();
+        sb.append("http://");
+        sb.append(host);
+        if (portNumber != null) {
+            sb.append(":").append(portNumber);
+        }
+        if (useDefaultUrlPath) {
+            sb.append("/zm");
+        } else if (urlPath != null) {
+            sb.append(urlPath);
+        }
+        sb.append(path);
+        return sb;
+    }
+
+    private boolean isMonitorIdValid(String id) {
+        return savedMonitors.stream().filter(monitor -> id.equals(monitor.getId())).findAny().isPresent();
+    }
+
+    private boolean isHostValid() {
+        logger.debug("Bridge: Checking for valid Zoneminder host: {}", host);
+        VersionDTO version = getVersion();
+        if (version != null) {
+            if (checkSoftwareVersion(version.version) && checkApiVersion(version.apiVersion)) {
+                return true;
+            }
+        } else {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR, "Can't get version information");
+        }
+        return false;
+    }
+
+    private boolean checkSoftwareVersion(@Nullable String softwareVersion) {
+        logger.debug("Bridge: Zoneminder software version is {}", softwareVersion);
+        if (softwareVersion != null) {
+            String[] versionParts = softwareVersion.split("\\.");
+            if (versionParts.length >= 2) {
+                try {
+                    int versionMajor = Integer.parseInt(versionParts[0]);
+                    int versionMinor = Integer.parseInt(versionParts[1]);
+                    if (versionMajor == 1 && versionMinor >= 34) {
+                        logger.debug("Bridge: Zoneminder software version check OK");
+                        return true;
+                    } else {
+                        logger.warn("Bridge: Requires version >= 1.34.0. This Zoneminder is version {}",
+                                softwareVersion);
+                        updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR,
+                                "Requires version >= 1.34.0");
```

Comment:
This is already logged by the framework via `updateStatus()`. You could remove the log message. Same for below.

Reply:
All logs removed